src/html/RD.jl

```diff
@@ -113,4 +117,59 @@ module RD
         ))
     end
     mathengine(::RequireJS, ::Nothing) = nothing
+
+    process_remote(dep, offline_version::Bool, build_path, origin_path=build_path) = offline_version ? _process(dep, build_path, origin_path) : dep
+    _process(dep::RemoteLibrary, build_path, origin_path) = RemoteLibrary(dep.name, _process(dep.url, build_path, origin_path); deps = dep.deps, exports = dep.exports)
+
+    function _download_file_content(url::AbstractString)
+        cmd = `curl $url -s --output -`
```

Comment:
Could also use Downloads.jl here. But `curl` is what we use elsewhere right now, and while we want to move away from it (https://github.com/JuliaDocs/Documenter.jl/issues/2254), its also completely fine to keep it as is.